vocab/types.json

```diff
@@ -1321,7 +1321,7 @@
         ]
     },
     "https://openminds.ebrains.eu/core/PropertyValueList": {
-        "description": null,
+        "description": "An identifiable list of property-value pairs.",
```

Comment:
"Structured information about an ..."

Reply:
The PropertyValueList, Configuration, QuantitativeValueX schemas are different from most others, in that they actually _contain_ the information. A PropertyValueList doesn't contain information about a list, it _contains the list_.

Reply:
Okay with me.